Docs/nav menu tooltip (#54)
siemens/ix-docs: docs/components/application-header/guide.md

- The brand logo disappears.
- The slot for additional elements (if used) moves to the overflow dropdown that opens on click on the overflow icon (13).

### Avatar

> With the new modular application frame we moved the avatar from the navigation menu to the application header. This ensures the avatar has security-relevant information available at all breakpoints.

Reviewer: ```suggestion
With the new modular application frame, we moved the avatar from the navigation menu to the application header. This ensures the avatar has security-relevant information available at all breakpoints.
```

![Avatar and avatar dropdown](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1635-60462&mode=design&t=UPXhDWuRHtygtfFI-11)

1. Avatar dropdown
2. User avatar
3. Top text line
4. Bottom text line

> The avatar is an optional element and indicates the current logged in user. If your application doesn’t support different users or user profiles, don’t use the avatar.

Author: unfortunately, we added it because we discovered that for some people, it's not 😅

> The avatar is an optional element and indicates the current logged in user. If your application doesn’t support different users or user profiles, don’t use the avatar.

Author: ```suggestion
The avatar is an optional element and indicates the current logged in user. If your application doesn’t support different users or user profiles, the avatar can be removed.
```
